templates/pay_from_payment_page.html.twig

```diff
@@ -19,7 +19,6 @@
             return fetch(createPayPalOrderFromPaymentPageUrl, {
                 method: 'post'
             }).then(function(res) {
-                document.forms[0].classList.add('loading');
```

Comment:
Would be great to have these scripts extracted into js instead of them being in the templates, but that's a topic for a later date.

Reply:
:+1: